.github/workflows/tests.yml

```diff
@@ -417,3 +417,108 @@ jobs:
           env_vars: TEST_SUBSET
           name: ${{ matrix.os }} ${{ matrix.floatx }}
           fail_ci_if_error: false
+  ubuntu_37:
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        floatx: [float64]
+        python-version: ["3.7"]
```

Comment:
You can just change to `python-version: ["3.x", "3.7"]` in all the jobs. If the goal is to remove support again immediately after, that will be make it an easier commit to review.

Reply:
You could have* (too late)

Reply:
we dont want to do that. We want to ensure the release works for all versions of PyMC that we want to support which includes 3.8, 3.9, 3.10 etc

Reply:
I think I didn't explain myself. Where you see e.g. `python-version=[3.8]` before, you could have changed to `python-version=[3.7, 3.8]` and both versions would have been tested. You could have done this for all existing tests, instead of copy the entirety of the first section of the file to just swap 3.8 to 3.7.

That would have been a much smaller PR and easier to review and revert. That's all

Reply:
that would have been way easier 🤦 

TIL. Thanks @ricardoV94